src/interface/get-order.interface.ts

```diff
@@ -0,0 +1,42 @@
+import { ICustomer } from '../interface';
+import { RetrieveInfo } from '../model';
+
+export interface IPayload {
+    customer: ICustomer;
+    info: RetrieveInfo;
+    items: IRetrieveItem[];
+    configuration: Record<string, any>;
+}
+
+export interface IRetrieveItem {
+    id: string;
+    'menu-item-name': string;
+    quantity: string;
+    price: string;
+    'special-instructions': string;
+    'menu-choices': string;
+    'menu-choices-hash': IMenuChoice;
+    'customer-name': string | null;
+}
+
+interface IChoice {
+    name: string;
+    price: number;
+}
+
+export type IMenuChoice = Record<string, IChoice[]>;
+
+export interface IGetOrder {
+    order: IRetrieveOrder;
+}
+
+export interface IRetrieveOrder {
+    guid: string;
+    payload: IPayload;
+    external_id: string;
+    created_at: string;
+    completed_at: string | null;
+    error_at: string | null;
+    error_string: string | null;
+    ticket_payload: null;
```

Comment:
maybe ticket_payload can be optional?

Reply:
as well as completed_at, error_at, error_string

Reply:
on. If no ticket applies will have a null value.
--

Reply:
![image](https://user-images.githubusercontent.com/34971977/82534207-6d7e4400-9b4d-11ea-8e4a-c7f596290787.png)

Reply:
does it mean `ticket_payload: string | null`?

Reply:
![image](https://user-images.githubusercontent.com/34971977/82534502-e8475f00-9b4d-11ea-98c7-467667bc627f.png)

Reply:
> does it mean `ticket_payload: string | null`?

 If no ticket applies will have a null value.

Reply:
From the first image that you sent `ticket_payload: string | null`